core/lib/multistore.ts

```diff
@@ -113,7 +113,7 @@ export function adjustMultistoreApiUrl (url: string) : string {
 }
 
 export function localizedRoute (routeObj: LocalizedRoute | string | RouteConfig | RawLocation, storeCode: string): any {
-  if (typeof routeObj === 'object' && routeObj && (<LocalizedRoute>routeObj).fullPath) return localizedDispatcherRoute(<LocalizedRoute>Object.assign({}, routeObj, { params: null }), storeCode)
+  if (typeof routeObj === 'object' && routeObj && (<LocalizedRoute>routeObj).fullPath && rootStore.state.config.seo.useUrlDispatcher) return localizedDispatcherRoute(<LocalizedRoute>Object.assign({}, routeObj, { params: null }), storeCode)
```

Comment:
`<LocalizedRoute>routeObj` -type is set 1 line above
`typeof routeObj === 'object' && routeObj` - isn't 1st check redundant?